apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -425,7 +434,16 @@ export function buildMachine(
         },
         on: {
           VOTER_VALIDATED_BALLOT: 'eject_to_rear',
-          VOTER_INVALIDATED_BALLOT: 'eject_to_front',
+          VOTER_INVALIDATED_BALLOT: 'invalidating_ballot',
+          NO_PAPER_ANYWHERE: 'resetting_state_machine_after_success',
+        },
+      },
+      // Ballot invalidation is a 2-stage process so the frontend can prompt the voter to get a pollworker
+      invalidating_ballot: {
```

Comment:
Nit: When I first read this state name, I thought maybe the backend was doing something to invalidate the ballot. Maybe would name this `waiting_for_invalidated_ballot_confirmation` or something.

Reply:
Thanks! [ac47292](https://github.com/votingworks/vxsuite/pull/3962/commits/ac472921d2a0ae8222d0827d0ab717ae927c667a)